project_forecast_line/tests/test_forecast_line.py

```diff
@@ -603,10 +603,150 @@ def test_task_forecast_lines_consolidated_forecast_overallocation_multiple_tasks
                 forecast1.employee_resource_forecast_line_id,
                 forecast2.employee_resource_forecast_line_id,
             )
-            self.assertEqual(
-                round(
-                    forecast1.employee_resource_forecast_line_id.consolidated_forecast,
-                    5,
-                ),
-                -0.75000,
+            self.assertAlmostEqual(
+                forecast1.employee_resource_forecast_line_id.consolidated_forecast,
+                -0.75,
+            )
+
+    def test_task_forecast_lines_employee_different_roles(self):
+        """
+        Test forecast lines when employee has different roles.
+
+        Employee has 2 forecast_role_id: consultant 75% and project manager 25%,
+        working 8h per day (standard calendar).
+        Create a task with forecast role consultant, with remaining time = 8h
+        and a scheduled period starting and ending on the same day (22/01/2022 for instance).
+        Assign this task to the user.
+
+        Expected: for the user, on 22/01/2022, 3 forecast lines.
```

Comment:
what matters are the values in these lines :)

Reply:
I updated with values, thanks for the help with this fix!